packages/firebase/README.md

```diff
@@ -177,7 +169,9 @@ $ npm install --save firebase
 In your code, you can access Firebase using:
 
 ```js
-var firebase = require('firebase');
+var firebase = require('firebase/app').default;
```

Comment:
I wanted to double check that Node needs the .default too? I vaguely remember maybe not?

Reply:
yes, if you want to get typings. Can you also update `var`s to `const`s?

Reply:
Updated.